### PR TITLE
Add ros dependencies to snap package too.

### PR DESCRIPTION
**Pull request by DavidMansolino**
**Description**
The snap package was missing the ros controller dependencies.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
I just tested the snap package created from this branch (https://github.com/cyberbotics/webots/releases/tag/untagged-164af96012d1f741e76f) and the ros controller is now working fine :tada:

**Review by omichel — APPROVED** [review]

File: scripts/packaging/webots_distro.c [hunk 1]
```diff
@@ -1191,50 +1252,13 @@ static void create_file(const char *name, int m) {
 #endif
 
       // add the required libraries in order to avoid conflicts on other Linux distributions
+      add_ros_dependencies("debian/usr/local/webots/");
```
Comment (omichel) [inline scripts/packaging/webots_distro.c:1255]:
```suggestion
      add_ros_dependencies("debian/usr/local/webots");
```

**Review by omichel — CHANGES_REQUESTED** [review]
One minor detail...

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**